PurdueHCR/ViewControllers/UserCreation/SignUpViewController.swift

```diff
@@ -99,6 +99,11 @@ class SignUpViewController: UIViewController, UITextFieldDelegate {
             self.signUpButton.isEnabled = true
             self.activityIndicator.stopAnimating()
         }
+		else if (password?.split(separator: " ").count != 1) {
```

Comment:
Fix spacing of code

Reply:
If the above conditional passes, will your code still be run?

Reply:
Spacing is correct inside of Xcode

Reply:
Since we know from the previous conditional that password has to not be empty, can you force unwrap instead of conditional unwrap? Just for clarity.